controllers/userController.js

```diff
@@ -8,6 +9,27 @@ exports.getAllUsers = async ctx => {
   // ctx.body= await db.User.findAll({attributes: ['username', 'password']})
   // (ctx.body = await dbAccess.getAllUsers())
 };
+
+
+module.exports.signIn = async (ctx, next) => {
+  const auth = ctx.request.headers.authorization.split(' ');
+  const decoded = atob(auth[1]).split(/:(.+)/);
+
+  console.log(decoded)
```

Comment:
console.log

Reply:
Well done! First comment I see so far in a review, for the entire seniors group!